boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -127,6 +127,47 @@
 	};
 };
 
+&octospi1 {
+	pinctrl-0 = <&octospi1_clk_pa3 &octospi1_ncs_pa2
+		     &octospi1_io0_pb1 &octospi1_io1_pb0
+		     &octospi1_io2_pa7 &octospi2_io3_pa6
+		     &octospi1_io4_pc1 &octospi1_io5_pc2
+		     &octospi1_io6_pc3 &octospi1_io7_pc0
+		     &octospi1_dqs_pb2>;
+	pinctrl-names = "default";
+
+	status = "okay";
+
+	mx25lm51245: ospi-nor-flash@0 {
+		compatible = "st,stm32-ospi-nor";
+		label = "MX25LM51245";
+		reg = <0>;
+		ospi-max-frequency = <50000000>;
+		size = <DT_SIZE_M(4)>;  /* 512 KBytes */
+		spi-bus-width = <8>;
+		data-rate = <2>;
```

Comment:
Can you use STM32_OSPI_OSPI_MODE and STM32_OSPI_STR_TRANSFER ?
Otherwise, not need to define them in a dt-binding header

Reply:
yes, 
OSPI / STR is a possible configuration
as well as OSPI/DTR and SPI/STR